frodo.js

```diff
@@ -10,27 +10,32 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.